### PR TITLE
Themes: Add screenshot to current theme bar

### PR DESCRIPTION
**Pull request by seear**
**Before**
screen shot 2017-03-08 at 19 41 15

**After**
screen shot 2017-03-08 at 19 37 37

**After Mobile**
screen shot 2017-03-09 at 14 37 05

Adds a small screenshot to the _Current Theme_ bar at wordpress.com/design/{site} to emphasise that it is displaying theme information.

**To Test**
Go to http://calypso.localhost:3000/design and select a site

### CODE REVIEW

**Comment by folletto** [comment]
Works well. Good idea 👍  

I'm not sure if it should be linkable, I'd launch not linked, as the links are just to the right of it. 

Also let's add a `1px` border left of the same weight of the `Card` border, so it doesn't bleed for white theme screenshots.

**Comment by seear** [comment]
@folletto Mobile view on the left or the right? Left is simplest implementation.

screen shot 2017-03-08 at 18 44 12

screen shot 2017-03-08 at 18 44 33

**Comment by seear** [comment]
>  Mobile view on the left or the right?

Or possibly hide on mobile to be less cluttered.

**Comment by folletto** [comment]
In my view: Right or Nothing. Slight preference for nothing. 👍

**Comment by drw158** [comment]
I like both options, left or right. Right looks a little bit better imo.

**Comment by seear** [comment]
> I like both options, left or right. Right looks a little bit better imo.

@drw158 I've gone with no screenshot at mobile width for now for simplicity of both looks and implementation, but happy to put it at the right if you feel strongly?

**Comment by drw158** [comment]
I think we should treat smaller viewports as first class! Because we can with not much effort, let's show the screenshot on mobile.

File: client/my-sites/themes/current-theme/index.jsx [hunk 1]
```diff
@@ -46,11 +46,19 @@ class CurrentTheme extends Component {
 			currentTheme && ! ( option.hideForTheme && option.hideForTheme( currentTheme, siteId ) )
 		);
 
+		const showScreenshot = currentTheme && currentTheme.screenshot;
+		// Show nothing for themes with no screenshot
+		const screenshotPlaceholder = ! currentTheme;
```
Comment (budzanowski) [inline client/my-sites/themes/current-theme/index.jsx]: maybe tie it to `showScreenshot` like 
```screenshotPlaceholder = ! showScreenshot```
and maybe change name to `showScreenshotPlaceholder` for consistency.

Comment (seear) [inline client/my-sites/themes/current-theme/index.jsx]: > maybe tie it to `showScreenshot` like
> `screenshotPlaceholder = ! showScreenshot`

Some (custom) themes have no screenshot, so the above line would leave a flashing placeholder. I'll improve the comment to try and make that clearer.

**Comment by ockham** [comment]
Quick after-the-fact comment -- I've noticed that on a JP site with a non-WP.com theme, the `?w=150` qs doesn't work, since `currentTheme.screenshot` contains a qs itself, `//ts.w.org/wp-content/themes/palm-beach/screenshot.jpg?ver=1.0.8`.

**Comment by seear** [comment]
Oh, nice spot. Same problem on the theme sheet:
screen shot 2017-03-10 at 14 39 30

**Pull request merged**